google-lb/pom.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.jclouds.examples</groupId>
+  <artifactId>google-lb</artifactId>
+  <version>1.9.0</version>
+  <name>google-lb</name>
+  <description>jclouds-labs-google example that shows using the compute specific api and constructing a load balancer.</description>
+
+  <properties>
+    <jclouds.version>1.9.0</jclouds.version>
+  </properties>
+
+  <dependencies>
```

Comment:
What do you think about leaving only the required dependencies to run the example? (the gce, jclouds-compute and ssh driver ones?).

Users might find easier to get started with their projects by just adding the "jclouds-all" dependency, but given that this example is GCE specific, it might make sense to have a small dependency set, showing how they can narrow the size of their projects. WDYT?